apps/web/src/components/SearchRequestTable/SearchRequestTable.tsx

```diff
@@ -87,8 +90,7 @@ export const SearchRequestTable = ({
           description:
             "Title displayed on the search request table requested date column.",
         }),
-        accessor: "requestedDate",
-        Cell: ({ value }) => dateAccessor(value, intl),
+        accessor: ({ requestedDate }) => dateAccessor(requestedDate, intl),
```

Comment:
I'm not clear why this is happening or even if it pre-existed but the the url search params for sort state are really nasty.  Looks like's it's encoding a JSON object.
![image](https://user-images.githubusercontent.com/8978655/217615607-103896fa-e9a2-4f21-8ce4-66c4d28d9521.png)

Reply:
That is exactly what it is doing 😜 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/518b6ee53199eed84530f000fada366eeec9ded1/apps/web/src/components/Table/ClientManagedTable/Table.tsx#L208

Reply:
Ah, maybe that's intended then.  :shrug: I guess I never noticed.

Reply:
Definitely ugly, I can't remember why but I did try to separate it out into each being a single value but it wasn't working. Maybe something we can revisit at some point 🤔